internal/security/secretstore/secure-messagebus.go

```diff
@@ -40,7 +40,7 @@ default:
   port: 6379
   protocol: redis
   server: localhost
-  serviceServer: http://localhost:48080
+  connectionSelector: edgex.redisMsgBus
```

Comment:
How is this related to the registry changes?

Reply:
This and the other two snap changes come from https://github.com/edgexfoundry/edgex-go/commit/85bf44e29f7317f38ef2a818ce6085ebcdec776e merged from main into this branch. I think they will be removed from the diff on squash+merge.

Reply:
@tonyespy , @farshidtz , Yep something went wrong on a rebase/merge and the PR picked up previous commits for previous PR that was already merged. Now fixed.